yaml_files/simphony_metadata.yml

```diff
@@ -170,6 +170,7 @@ CUDS_KEYS:
     CUBA.RHEOLOGY_MODEL:
       default: CUBA.NEWTONIAN_FLUID_MODEL
     CUBA.GRAVITY_MODEL:
+      scope: CUBA.USER
```

Comment:
The quick solution is putting `default: `. It will pass `None` as default value.